.github/CONTRIBUTING.md

```diff
@@ -22,6 +22,7 @@ Brave welcomes contributions via [pull requests](https://github.com/brave/browse
     Fix #206
 ````
 
+* If you update the npm dependencies, run `npm shrinkwrap --dev` before committing changes.
```

Comment:
@luixxiul we'll want to pick and choose the times we update dependencies. Unless there is a good reason (need new version to get a feature, we're too out of date, etc) , we should stick with the version there.  This is important to stabilize the codebase.  

Any updates we do to libraries we use may require a small regression test to ensure the upgrade didn't break any functionality or introduce issues